doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -256,6 +256,8 @@ Wi-Fi
 -----
 
 * The WPA supplicant is now switched to Zephyr upstream's fork instead of |NCS|.
+* Added the :ref:`ug_nrf70_developing_offloaded_raw_tx` page.
+* Added support advanced security modes in the Wi-Fi stack, please see the :ref:`ug_nrf70_wifi_advanced_security_modes` page for more information.
```

Comment:
```suggestion
* Added:

  * The :ref:`ug_nrf70_developing_offloaded_raw_tx` page.
  * Support for :ref:`ug_nrf70_wifi_advanced_security_modes` in the Wi-Fi stack.

* Updated:

  * The WPA supplicant is now switched to Zephyr upstream's fork instead of |NCS|.
```